cluster-provision/k8s/1.19/provision.sh

```diff
@@ -59,54 +69,33 @@ yum -y install iscsi-initiator-utils
 # To prevent preflight issue realted to tc not found
 dnf install -y tc
 
-# Install docker required packages.
-dnf -y install yum-utils \
-    device-mapper-persistent-data \
-    lvm2
-
-# Add Docker repository.
-dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+export CRIO_OS=CentOS_8_Stream
+export CRIO_VERSION=1.19
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$CRIO_OS/devel:kubic:libcontainers:stable.repo
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:$CRIO_VERSION.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$CRIO_VERSION/$CRIO_OS/devel:kubic:libcontainers:stable:cri-o:$CRIO_VERSION.repo
+dnf install -y cri-o
 
-# Enable the nightly version to get a new enough runc version which includes containerd
-# with a new enough runc version to avoid https://github.com/kubernetes/kubernetes/issues/95296#issuecomment-714419461
-# which manifests in /dev/* permission denied flakes on cpu manager enabled nodes
-dnf config-manager --set-enabled docker-ce-nightly
+# Install crictl
+(
```

Comment:
since you use sub shell here, please add -E to the set, else it wont stop on errors

Reply:
Done